src/components/RepoComponent.js

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useState } from 'react';
+import Accordion from 'react-bootstrap/Accordion';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+
+import {fetchRepos} from "../services/fetchTeamDetails"
+import "./repoComponent.css"
+
+const RepoComponent=({repoUrl})=> {
+    
+   const [repoDetails, setRepoDetails] = useState([]);
+    
+    useEffect(()=>{
+        repoUrl && fetchRepos(repoUrl).then((repo)=>{
+            setRepoDetails(repo);
```

Comment:
No need of repoUrl here the setrepoDetails will be set once repoUrl is available.

U were facing any issues? Why u used repoUrl && ?